### PR TITLE
[REVERT] Add back Event Editable frontend

### PR DESCRIPTION
**Pull request by yicyao**
Exact same changes as: Event editable [Frontend] #98
Changes lost in merge of backend (branched off there), all changes were originally already reviewed in #98 (but then changes all lost) . Build fails because no backend files.

Add edit ability to events after you create them. Redirects to the create event form with the previous form information that you can now change and write back to the db.

### CODE REVIEW

File: project/src/main/webapp/event-edit.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                aria-describedby="basic-addon1"
+              />
+            </div>
+          </div>
+        </div>
+        <br />
+        <div class="inline">
+          <text> Tags: </text>
+          <div class="container">
+            <span class="badge badge-primary text-medium"> Tag 1</span>
```
Comment (dash102) [inline project/src/main/webapp/event-edit.html:86]: This part at least looks like an older version - newer version of event edit should include the updated interests and skills section

Comment (yicyao) [reply project/src/main/webapp/event-edit.html:86]: ty for looking so carefully! will fix

File: project/src/main/webapp/event-edit.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+      integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body onload = "getEventDetails()">
```
Comment (nikhitashanker) [inline project/src/main/webapp/event-edit.html:27]: It would be better to move the onload functions to the javascript file to keep style consistent.

File: project/src/main/webapp/create-event.html [hunk 3]
```diff
@@ -52,131 +56,124 @@ <h3>Create an Event</h3>
...
-            ></select>
-            <button 
-              type="button"
-              class="btn btn-light prefilled-pick"
-              onclick="togglePrefilledSkills();"
-            >+ Pick</button>
-            <div id="prefilled-skills" class="overflow-auto prefilled-pool"></div>
-          </div>
-        </div>
+      <div class="col-lg-6">
```
[the remark this answers is not in the document]
Comment (yicyao) [reply project/src/main/webapp/create-event.html:89]: Thanks for noticing! I reverted changes from before that was piushed to master. Added back.

File: project/src/main/webapp/event-details.js [hunk 4]
```diff
@@ -91,23 +91,23 @@ function getLinkForOpportunity(opportunityId) {
...
+  const response = await fetch('/create-event?' + new URLSearchParams({'eventId': eventId}))
+  const data = await response.json();
+  document.getElementById('name').innerHTML = data['name'];
+  document.getElementById('description').innerHTML = data['description'];
+  document.getElementById('date').innerHTML = `Date: 
+  ${data['date'].month}/${data['date'].dayOfMonth}/${data['date'].year}`;
+  document.getElementById('location').innerHTML =
+    `Location: ${data['location']}`;
+  document.getElementById('time').innerHTML = `Time: ${data['time']}`;
+  const link = '/event-edit.html?eventId=' + eventId;
```
[the remark this answers is not in the document]
Comment (yicyao) [reply project/src/main/webapp/event-details.js:109]: Done for 109, not doing it on line 100 bc java backend looks for params posted.

File: project/src/main/webapp/event-edit.html (hunk 1 above)
Comment (yicyao) [reply project/src/main/webapp/event-edit.html:86]: Could you link me to a PR / file where this is done?

File: project/src/main/java/com/google/sps/utilities/SpannerTasks.java [hunk 5]
```diff
@@ -112,7 +112,7 @@ public static void insertOrUpdateUser(User user) {
    *
    * @param event the event to be inserted or updated; event's ID field should not exist in DB
    */
-  public static void insertorUpdateEvent(Event event) {
+  public static void insertOrUpdateEvent(Event event) {
     List<Mutation> mutations =
```
Comment (nikhitashanker) [inline project/src/main/java/com/google/sps/utilities/SpannerTasks.java:116]: Please revert this change I wrote tests with the spelling insertorUpdate so those will fail.

File: project/src/main/webapp/event-edit.html (hunk 1 above)
Comment (dash102) [reply project/src/main/webapp/event-edit.html:86]: You can look at create-event.html for examples of how it looks like rn
